user-tools/README-CVImage.md

```diff
@@ -14,17 +43,19 @@
 
 3. Create a new bot segment json file and copy the base64 encoded output as the `imageData` of your CVImage criteria. The `withinRect` field is optional and can be used to limit the search to specific regions of the screen.  Be aware that the `rect` field is defined in coordinates where the `x` and `y` coordinates are 0 at the bottom left of the screen.  The `screenSize` field defines the overall coordinate space for the specified `rect` and is used to linearly transform the `rect` into the current game resolution during runtime analysis thus suporting varying resolutions.
 
+## Example Segment JSON
+
+### `endCriteria`
+
 ```json
 {
+    "name":"CV Image Criteria: Menu Change Profile Button",
     "sessionId":"12345",
```

Comment:
I feel like if we are not documenting sessionID, we shouldn't give it in the examples either. (or we should document it)